### PR TITLE
RET-3910

### PR DESCRIPTION
**Pull request by GauravChawlaMOJ**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RET-3910

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: definitions/json/AuthorisationCaseField.json [hunk 1]
```diff
@@ -2423,6 +2423,54 @@
...
+  {
+    "CaseTypeId": "ET_Scotland",
+    "CaseFieldID": "claimantDocumentCollection",
+    "UserRole": "caseworker-employment-etjudge",
+    "CRUD": "R"
+  },
+  {
+    "CaseTypeId": "ET_Scotland",
+    "CaseFieldID": "claimantDocumentCollection",
+    "UserRole": "caseworker-employment-englandwales",
```
Comment (hsjhita) [inline definitions/json/AuthorisationCaseField.json:2447]: Scotland and same for line 2459

File: definitions/json/AuthorisationCaseField.json [hunk 2]
```diff
@@ -2423,6 +2423,54 @@
     "UserRole": "et-acas-api",
     "CRUD": "R"
   },
+  {
+    "CaseTypeId": "ET_Scotland",
+    "CaseFieldID": "claimantDocumentCollection",
+    "UserRole": "caseworker-employment-legalrep-solicitor",
+    "CRUD": "CRU"
```
Comment (hsjhita) [inline definitions/json/AuthorisationCaseField.json:2430]: Maybe should be R only for the legal rep

Comment (GauravChawlaMOJ) [reply definitions/json/AuthorisationCaseField.json:2430]: done

File: definitions/json/AuthorisationCaseField.json (hunk 1 above)
Comment (GauravChawlaMOJ) [reply definitions/json/AuthorisationCaseField.json]: changed

**Pull request merged**